### PR TITLE
Workflows consistent with readme

### PR DESCRIPTION
**Pull request by jspaaks**
replaced the docker commands in the workflow files to docker-compose commands, to be consistent with what is described in the readme.

### CODE REVIEW

File: model-generation/environment.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - ipykernel
   - jupyter
   - nbconvert
+  - pandas
```
Comment (fdiblen) [inline model-generation/environment.yml:11]: Why did you add pandas again?

**Review by fdiblen — CHANGES_REQUESTED** [review]
Thank you @jspaaks 
I have a single question.

**Review by fdiblen — APPROVED** [review]
Dock dock docker.
:dolphin:

File: model-generation/environment.yml (hunk 1 above)
Comment (jspaaks) [reply model-generation/environment.yml:11]: it was needed by some other PR (the progress bar one) For reference, https://github.com/ADAH-EviDENce/evidence/pull/92#issuecomment-661086019.
